crates/driver/src/tests/setup/blockchain/mod.rs

```diff
@@ -33,17 +33,30 @@ impl Geth {
     }
 }
 
+// What we really want here is "AsyncDrop", which is an unsolved problem in the
+// async ecosystem. As a workaround we create a new runtime so that we can block
+// on the delete request. Spawning a task for this isn't enough because tokio
+// runtimes when they exit drop background tasks, like when a #[tokio::test]
+// function returns.
```

Comment:
I think this comment makes sense in the context of the PR but contains a little too much information on its own.

```suggestion
/// What we really want here is "AsyncDrop", which is an unsolved problem in the
/// async ecosystem. As a workaround we create a new runtime so that we can block
/// on the delete request.
```

Reply:
The reason I included that extra sentence is that I feel someone reading this later is going to be confused why we not just spawn a task. They could find the original commit that made the change but that can be tedious. I'm fine with either wording.